docs/pages/workflow/expo-cli.md

```diff
@@ -542,7 +542,7 @@ Alias: `expo ba`
 | `--generate-keystore`              | [deprecated] Generate Keystore if one does not exist            |
 | `--public-url [url]`               | The URL of an externally hosted manifest (for self-hosted apps) |
 | `--skip-workflow-check`            | Skip warning about build service bare workflow limitations.     |
-| `-t --type [build]`                | Type of build: [app-bundle                                      | apk]. |
```

Comment:
This is autogenerated in https://github.com/expo/expo-cli/blob/master/packages/expo-cli/scripts/introspect.ts, so any manual changes will simply be overwritten.